### DISCUSSION TITLE
List Groups

### DISCUSSION THREAD

**Discussion by kimfucious** [mantinedev/mantine#1794] [Feature requests]
I've just discovered you by fluke, and I gotta say, I'm loving Mantine!  I've just adopted it for a new project I'm working on.  So thanks.

Your docs are great, and the products are super-duper.

There is one thing, however, that I think would be a great thing to add, unless there's an easy way to accomplish this with the existing suite of components: [List Groups](https://getbootstrap.com/docs/5.0/components/list-group/).

I hate to even mention the name, He Who Shall Not be Named(a.k.a. Bootstrap), but that component is a really nice piece of work.

I've tried to map a bunch of Groups under a Stack to recreate what list groups do, somewhat successfully, but it's somewhat tedious.

If you folks could put such a thing in your backlog, that would be awesome.

Keep up the great work!

**Comment by muco-rolle** [comment]
Go through https://ui.mantine.dev/ especially:

- https://ui.mantine.dev/category/tables
- https://ui.mantine.dev/category/inputs

 you will find the same things as the List Group

**Reply by kimfucious** [reply]
Thanks for the reply @muco-rolle.

While I did manage to cobble together something like a list group, my main reason for suggesting this as a feature was to not have to do any cobbling.

Really enjoying Mantine!
